test_regress/t/t_assert_past.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003-2009 by Wilson Snyder. This program is free software; you
```

Comment:
```suggestion
# Copyright 2022 by Antmicro Ltd. This program is free software; you
```
Same for the other test files.

Reply:
I've modified the licenses. When it comes to compatibility with V5 - I think the effort to glue it will be significant, so I'm looking for another pass that could contain that code.